app/service/parsing_svc.py

```diff
@@ -1,14 +1,48 @@
-import plugins.stockpile.parsers.standard as parsers
-import plugins.stockpile.parsers.mimikatz as mimikatz_parser
+import glob
 from base64 import b64decode
+from pydoc import locate
 
 from app.service.base_service import BaseService
 
 
 class ParsingService(BaseService):
 
     def __init__(self):
+        self.parsers = dict()
         self.log = self.add_service('parsing_svc', self)
+        
+    async def load_parser(self, p_id, name, parser):
+        """
+        Load the specified parser plugin
+        :param p_id:  the id of the yaml file that defines the parser
+        :param name:  the unique name of the parser
+        :param parser:  an instance of the parser class 
+        :return: None
+        """        
+        if name in self.parsers:
+            self.log.warning('Duplicate parser name detected:  %s:%s' % (p_id, name))
+        else:
+            self.parsers[name] = parser       
+        
+    async def load_parsers(self, directory):
+        """
+        Load all parser plugins
+        :param directory:
```

Comment:
docstring

Reply:
This appears to be the convention used in the class already.  I just followed what was already there.  Please clarify what change you are requesting.

Reply:
"do as the locals do even if what the locals do is stupid"
Ideally docstrings would be filled out, but leave as is for consistency